addons/web/static/src/webclient/settings_form_view/settings_form_view_mobile.scss

```diff
@@ -63,4 +63,4 @@
         }
 
     }
-}
```

Comment:
Hello !
Care the end of file line, asides from that I like the visual result just maybe a bit difficult to spot the scroll when we are in the general setting page because of the long name but since it doesn't affect the scrolling area it's not really a problem